core/src/main/java/me/mykindos/betterpvp/core/client/repository/ClientRepository.java

```diff
@@ -38,7 +38,7 @@ public ClientRepository(Database database) {
     @Override
     public List<Client> getAll() {
         List<Client> clients = new ArrayList<>();
-        String query = "SELECT * FROM clients;";
+        String query = "SELECT * FROM " + databasePrefix + "clients;";
```

Comment:
Because some of these core tables are potentially used across multiple servers, the prefix wont work as expected.

A much larger rework will likely be required